keyboards/rpiguy9907/southpaw66/readme.md

```diff
@@ -0,0 +1,24 @@
+# Southpaw66
+
+![](https://i.imgur.com/Mzmhsedl.jpg)
+
+### Pro Micro Pinout
+| rows            | 0   | 1   | 2   | 3   | 4   | 5   | 6   |
+| --------------- | --- | --- | --- | --- | --- | --- | --- |
+| Pro Micro pin   | D3  | D2  | D1  | D0  | D4  | C6  | D7  |
+
+| columns         | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   |
+| --------------- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| Pro Micro pin   | E6  | B4  | B5  | F4  | F5  | F6  | F7  | B1  | B3  | B2  |
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make rpiguy9907/southpaw66:default
+
+**Reset Key:** Hold down the key located at `k00`, commonly programmed as Escape while plugging in the keyboard.
```

Comment:
1. Bootmagic Lite is enabled, but the key isn't changed from the default of `row 0, col 0`, which on this board is Delete rather than Escape.

```suggestion
**Reset Key:** Hold down the key located at `k10`, commonly programmed as Escape while plugging in the keyboard.
```